### DISCUSSION TITLE
when you divide document at this page, why does the file size remain the same?

### DISCUSSION THREAD

**Discussion by nxnode** [RyotaUshio/obsidian-pdf-plus#177] [Q&A]
I am working with a 500MB pdf that has about 300 pages. After breaking the pdf into smaller pieces (using the "divide document at this page" feature) I discovered all of the smaller pdf's are 500MB. Even the pdf's that are only 2 or three pages - they all show up in the vault as 500MB. Is this normal?

**Comment by RyotaUshio** [comment]
Thanks for letting me know about this. This seems to be an upstream problem of PDF-LIB, a JavaScript library that PDF++ uses to manipulate PDF files.

https://github.com/Hopding/pdf-lib/issues/140

I'll look for workarounds and update the docs etc.
